src/reducers/scrollAnchor.js

```diff
@@ -74,6 +74,19 @@ export function scrollTo(state, scrollY) {
   const { rowOffsetIntervalTree } = state.getInternal();
   const { rowsCount } = rowSettings;
 
+  if (state.rowSettings.rowHeightGetter != undefined) {
+    // In case of variable row height, ask for the actual heights of the rows before the scroll position.
+    // Only for the ones that were not asked before
+    let { rowUntilOffsetsAreExact, rowOffsetIntervalTree } =
+      state.getInternal();
+
+    let exactHeight = rowOffsetIntervalTree.sumUntil(rowUntilOffsetsAreExact);
+    while (exactHeight < scrollY && rowUntilOffsetsAreExact < rowsCount) {
+      exactHeight += updateRowHeight(state, rowUntilOffsetsAreExact++);
+    }
+    state.rowUntilOffsetsAreExact = rowUntilOffsetsAreExact;
```

Comment:
this wont work because rowUntilOffsetsAreExact is not a property of the state but a property of the internalState in the state. 
So what we can instead do is 
`state.getInternal().rowUntilOffsetsAreExact = rowUntilOffsetsAreExact;`

Reply:
Indeed. I have modified as suggested. And I have added a test for checking that the mechanism related to rowUntilOffsetsAreExact is working ok